deploy/helm/kubecf/values.yaml

```
@@ -47,7 +47,7 @@ releases:
   bits-service:
     version: 2.28.0
   brain-tests:
-    version: v0.0.8
+    version: v0.0.9
```

I believe this (for now) needs a stemcell override:
```suggestion
    version: v0.0.9
    stemcell:
      os: SLE_15_SP1
      version: 23.11-7.0.0_374.gb8e8e6af
```

---

Ok.